lib/tasks/globalize.rake

```diff
@@ -0,0 +1,84 @@
+namespace :globalize do
+  def translatable_classes
+    [
+      AdminNotification,
+      Banner,
+      Budget::Investment::Milestone,
+      I18nContent,
+      Legislation::DraftVersion,
+      Legislation::Process,
+      Legislation::Question,
+      Legislation::QuestionOption,
+      Poll,
+      Poll::Question,
+      Poll::Question::Answer,
+      SiteCustomization::Page,
+      Widget::Card
+    ]
+  end
+
+  def migrate_data
+    @errored = false
+
+    translatable_classes.each do |model_class|
+      logger.info "Migrating #{model_class} data"
+
+      fields = model_class.translated_attribute_names
+
+      model_class.find_each do |record|
+        fields.each do |field|
+          locale = if model_class == SiteCustomization::Page && record.locale.present?
+                     record.locale
+                   else
+                     I18n.locale
+                   end
+
+          if record.send(:"#{field}_#{locale}").blank?
+            record.send(:"#{field}_#{locale}=", record.untranslated_attributes[field.to_s])
```

Comment:
Hi @javierm! I was just catching up on the latest "translatable content" progress and noticed a potential bug here (I could be wrong, I haven't looked at the code for the past month).

Basically what I describe in more detail [here](https://github.com/consul/consul/pull/2914#discussion_r219226899): this direct interpolation is not going to work for locales with uppercase letters or hyphens, such as `pt-BR`, because those characters are not allowed in Ruby method names.

Reply:
Thank you so much for this comment! I'll have a look and add a test.

Reply:
Thank you so much for this comment! I'll have a look and add a test.

Reply:
Thank you so much for this comment! I've added a test and used your suggestion to fix the bug.

Reply:
Thank you so much for this comment! I've added a test and used your suggestion to fix the bug :+1:.

Reply:
Thank you so much for this comment! I've added a test and used your suggestion to fix the bug :+1:.

Reply:
Thank you so much for this comment! I've added a test for this case and applied your suggestion to fix it :+1:.